ts/defs.bzl

```diff
@@ -318,19 +325,43 @@ def ts_project(
     typings_out_dir = declaration_dir if declaration_dir else out_dir
     tsbuildinfo_path = ts_build_info_file if ts_build_info_file else name + ".tsbuildinfo"
 
-    tsc_typings_outs = _lib.calculate_typings_outs(srcs, typings_out_dir, root_dir, declaration, composite, allow_js, no_emit)
-    tsc_typing_maps_outs = _lib.calculate_typing_maps_outs(srcs, typings_out_dir, root_dir, declaration_map, allow_js, no_emit)
+    # Target names for tsc, dts+js transpilers
+    tsc_target_name = name
+    declarations_target_name = None
+    transpile_target_name = None
+
+    # typing outputs
+    tsc_typings_outs = []
+    tsc_typing_maps_outs = []
+    if not declaration_transpiler or declaration_transpiler == "tsc":
```

Comment:
Don't need to have the explicit `tsc` option for `declaration_transpiler` we do for `transpiler`.

```
if not declaration_transpiler:
```

should be sufficient since we're not forcing users to either set `--@aspect_rules_ts//ts:default_to_tsc_transpiler` or an explicit `tsc` string.

Reply:
That sounds right, I'll remove the magic "tsc" on this line. I'm not sure why I originally did that (this PR has been through a few iterations now...).